builder/rootfs/bin/boot

```diff
@@ -74,6 +88,27 @@ done
 docker build -t deis/slugbuilder /usr/local/src/slugbuilder/
 docker build -t deis/slugrunner /usr/local/src/slugrunner/
 
+function gen_host_keys {
+	# generate the keys, then set them up in etcd if not already set
+	/usr/bin/ssh-keygen -A
```

Comment:
it's a little unnecessary, though this was the most efficient way I could think of to generate the keys without introducing too much branch complexity. If you'd prefer to save a few milliseconds, I can wrap this and check if one of the keys have already been saved in etcd. Would that be ok?